### ISSUE TITLE
bug: 5.3.0 initialisation issue

### ISSUE THREAD

**Issue by pritamsoni-hsr** [ionic-team/ionic-framework#21803]
# Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.3.0**

**Current behavior:**

[video](https://youtu.be/sBo89B2tH64)

**Expected behavior:**

**Steps to reproduce:**

just updated to 5.3.0 and got this error everything was working fine in 5.2.3

**Related code:**

```sh
npm update @ionic/react
npm update @ionic/react-router
```

**Other information:**

**Ionic info:**

```
Ionic:

   Ionic CLI       : 6.10.0 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework : @ionic/react 5.3.0
```

**Comment by pritamsoni-hsr** [comment]
@liamdebeasi just create a react project from scratch and update ionic react and router from 5.0.7 to 5.3, this is issue will come up, or maybe this could be specific to mac.

**Comment by liamdebeasi** [comment]
I am testing the Ionic React tabs starter application on macOS with Ionic Framework 5.3.0 and am not running into any issues. Can you please provide a full repo with the code required to reproduce this issue?

**Comment by pritamsoni-hsr** [comment]
[here](https://github.com/pritamsoni-hsr/ionic-test) is the repo,
I'm using Node v14.4.0, npm 6.14.4, ionic cli 6.10.0,  macOS Catalina 10.15.5.

**Comment by liamdebeasi** [comment]
I am not running into any issues with that repo. Try removing `node_modules` and doing a fresh `npm install`.

**Comment by pritamsoni-hsr** [comment]
@liamdebeasi I did a fresh reinstall, still facing the same issue.

**Comment by liamdebeasi** [comment]
Do you have the latest `@types` packages?

**Comment by pritamsoni-hsr** [comment]
```sh
"@types/react": "^16.9.17",
"@types/react-router": "^5.1.4",
```
never needed for ionic (react and router) tho, it's already TS based.

**Comment by liamdebeasi** [comment]
I had another colleague try your repo and they were unable to reproduce the issue either. Is there any other information you can give us to help us reproduce the issue?

**Comment by pritamsoni-hsr** [comment]
not working i tried again cloning and `npm i`, probably its "not on my machine" issue, I'll try docker or something.
Are you using Ionic CLI v 6.10.0?

**Comment by liamdebeasi** [comment]
Yes, I am testing with Ionic CLI v6.10.0.

**Comment by pritamsoni-hsr** [comment]
facing same issue with docker.

**Comment by liamdebeasi** [comment]
Can you provide any other steps we can use to reproduce this issue? Otherwise I am going to have to close the issue since we cannot reproduce.

Can you try cloning this app and see if you run into any build issues? https://github.com/ionic-team/ionic-conference-app

**Comment by liamdebeasi** [comment]
Hey there! We just reproduced the issue. Looks like it only reproduces in Safari. I was testing in Chrome which explains why I could not reproduce the issue. We will look into this.

**Comment by pritamsoni-hsr** [comment]
Thanks, how could I forget to mention that.

**Comment by liamdebeasi** [comment]
Thanks! This was resolved via https://github.com/ionic-team/ionic-framework/issues/21803, and a fix will be available in the next release of Ionic Framework.

**Comment by liamdebeasi** [comment]
Hey there, we have released this fix as part of Ionic Framework v5.3.1.

Upgrade Steps:
```
npm install @ionic/react@5.3.1 @ionic/react-router@5.3.1
```
